Dockerfile

```diff
@@ -66,8 +66,8 @@ RUN \
   apt install -y --no-install-recommends python2 python-pip python-setuptools \
   build-essential libxml2-dev libxslt-dev python-dev zlib1g-dev && \
   pip2 install wheel && \
-  pip2 install -r requirements.txt 
-
+  pip2 install -r requirements.txt && \
+  find /usr/lib -iname "libffi.so.6*" -exec cp {} /usr/local/lib \;
```

Comment:
Could a comment be added about this? Like which piece/version needs it, and how to tell when some day it may no longer be needed.

Reply:
Comment added.